contracts/azorius/LinearERC20Voting.sol

```diff
@@ -153,17 +171,13 @@ contract LinearERC20Voting is BaseStrategy, BaseQuorumPercent {
 
     /** @inheritdoc IBaseStrategy*/
     function isPassed(uint256 _proposalId) public view override returns (bool) {
-        if (
-            proposalVotes[_proposalId].yesVotes > proposalVotes[_proposalId].noVotes &&
-            proposalVotes[_proposalId].yesVotes >=
-            quorum(proposalVotes[_proposalId].votingStartBlock) &&
-            proposalVotes[_proposalId].votingEndBlock != 0 &&
-            block.number > proposalVotes[_proposalId].votingEndBlock
-        ) {
-            return true;
-        }
-
-        return false;
+        return (
+            proposalVotes[_proposalId].votingEndBlock != 0 && // end block wasn't set to 0 TODO why do we need this?
```

Comment:
@tbwebb22 is it necessary to check `votingEndBlock != 0` here?

Reply:
That check is to ensure that the proposal has actually been submitted

Reply:
However, our new basis votes check may ensure that proposals that haven't been submitted don't return true for this

Reply:
This is resolved, the new `isPassed` logic handles proposals that haven't been submitted yet.